Build failure with MSVC in platform/iphone/export/export.cpp: 'chmod': identifier not found

**Operating system or device - Godot version:**
Windows 10 with VS2015

**Issue description:**
platform\iphone\export\export.cpp(265): error C3861: 'chmod': identifier not found
scons: *** [platform\iphone\export\export.windows.tools.64.obj] Error 2
scons: building terminated because of errors.

--- a/platform/iphone/export/export.cpp
+++ b/platform/iphone/export/export.cpp
@@ -260,10 +260,12 @@ Error EditorExportPlatformIOS::export_project(const Ref<EditorExportPreset> &p_p
 			f->close();
 			memdelete(f);
 
+#ifdef OSX_ENABLED
 			if (is_execute) {
 				// we need execute rights on this file
 				chmod(file.utf8().get_data(), 0755);
 			}
+#endif
 		}
 
 		ret = unzGoToNextFile(src_pkg_zip);

